ios/utils/RNSDefines.h

```diff
@@ -5,3 +5,18 @@
       _Pragma("clang diagnostic ignored \"-Wobjc-missing-super-calls\"")
 
 #define RNS_IGNORE_SUPER_CALL_END _Pragma("clang diagnostic pop")
+
+#if defined __has_include
+#if __has_include( \
+    <React-RCTAppDelegate/RCTReactNativeFactory.h>) // added in 78
+#define RNS_REACT_NATIVE_VERSION_MINOR_BELOW_78 0
+#else
+#define RNS_REACT_NATIVE_VERSION_MINOR_BELOW_78 1
+#endif
+#endif
```

Comment:
This is not pretty, but good enough for now. 

If we need to have more RN version dependent code in the future we should define `RNS_REACT_NATIVE_VERSION_MINOR` during compilation or something. 

I've also seen that `REACT_NATIVE_VERSION_MINOR` macro is defined in `ReactNativeVersion.h` since RN 0.79.